src/Angor/Client/Pages/View.razor

```diff
@@ -565,8 +565,7 @@
                         var invpub = (project as InvestorProject).InvestorNPub;
                         var invNpub = NostrHelper.ConvertHexToNpub(invpub);
 
-                        <br />  <br />
-                        <label for="NostrNpubPublicKey" class="form-label">Investor NOSTR public key (npub)</label>
+                        <label for="NostrNpubPublicKey" class="form-label mt-6">Investor NOSTR public key (npub)</label>
```

Comment:
you removed the <br /> <br /> dont you think we should make a separation?

Reply:
I used mt-6 class instead of br tag